Cue/app/api/Authentication.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import { serverURL } from '../env';
+
+var AuthenticationApi = {
+	authenticate(userId: ?string, accessToken: ?string) {
+		let url = serverURL+'/api/v1/auth';
+		let method = 'POST'
+		let body = JSON.stringify({
+				user_id: userId,
+				access_token: accessToken,
+			})
+
+		return fetch(url, {
+			method: method,
+			body: body
+		})
+		.then(checkStatus)
+		.then(response => response.json())
+		.catch(e => {
+			throw (e)
+		});
+	},
+}
+
+var checkStatus =function(response) {
+	if (response.status >= 200 || response.status <= 300) {
```

Comment:
This should be `&& response.status < 300` instead of `|| response.status <= 300`